fix gcc compilation

just fix compilation under gcc.
Upstream-Status: Submitted [https://github.com/microsoft/MSRSec/pull/35]
Compilation issue already fixed in wolfcrypt and submodule
needs to be updated. I sent PR for for update with the link
above.

Signed-off-by: Maxim Uvarov <maxim.uvarov@linaro.org>

diff --git a/wolfssl/wolfcrypt/types.h b/wolfssl/wolfcrypt/types.h
index 7b3a953aebda..e156ae5c7909 100755
--- a/external/wolfssl/wolfssl/wolfcrypt/types.h
+++ b/external/wolfssl/wolfssl/wolfcrypt/types.h
@@ -181,7 +181,7 @@
     /* GCC 7 has new switch() fall-through detection */
     #if defined(__GNUC__)
         #if ((__GNUC__ > 7) || ((__GNUC__ == 7) && (__GNUC_MINOR__ >= 1)))
-            #define FALL_THROUGH __attribute__ ((fallthrough));
+            #define FALL_THROUGH __attribute__ ((__fallthrough__));
         #endif
     #endif
     #ifndef FALL_THROUGH
